cmd/aro/update_ocp_versions.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd"
+	"github.com/Azure/ARO-RP/pkg/util/encryption"
+	"github.com/Azure/ARO-RP/pkg/util/keyvault"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShiftVersion, error) {
+	env, err := env.NewCoreForCI(ctx, log)
+	if err != nil {
+		return nil, err
+	}
+	dstAcr := os.Getenv("DST_ACR_NAME")
+	acrDomainSuffix := "." + env.Environment().ContainerRegistryDNSSuffix
+
+	dstRepo := dstAcr + acrDomainSuffix
+	var (
+		OpenshiftVersions = []api.OpenShiftVersion{
+			{
+				Version:           version.InstallStream.Version.String(),
+				OpenShiftPullspec: version.InstallStream.PullSpec,
+				InstallerPullspec: dstRepo + "/aro-installer:release-4.10",
```

Comment:
Hard coded value required here?

Reply:
Yep, we are temporarily hard coding the releases to get this functionality into the next release. But we will be retrieving the versions from the Cincinnati stable channel at a later stage